### DISCUSSION TITLE
Suggest to reduce the space occupied by a single list

### DISCUSSION THREAD

**Discussion by yqs112358** [community/community#9270] [Other Feature Feedback, Questions, & Ideas]
It seems that when I set up numbers of star list, it takes too much space at the top of the page, because that only one list exists in one line. 
<img width="512" alt="image" src="https://user-images.githubusercontent.com/37969157/147313203-aab73de8-b8dd-4369-a3e4-df000a6ceac1.png">

In my point of view, a better UI design will be like which your account's pinned repository shows: Compact buttons, allow you to see multiple options at one glance
<img width="512" alt="image" src="https://user-images.githubusercontent.com/37969157/147313497-e6be1b76-101a-4597-bde7-3e4e2e6ace3b.png">

**Comment by brianlovin** [comment]
Thanks for the suggestion @yqs112358! I'll pass this along to the team to think of better ways to order/group/render lists so that they use the screen space more efficiently 😊
